ts/screens/profile/LanguagesPreferencesScreen.tsx

```diff
@@ -124,8 +127,18 @@ const LanguagesPreferencesScreen = () => {
       !pot.isError(profile)
     ) {
       setIsLoading(false);
-      preferredLanguageSaveSuccessDispatch(selectedItem as Locales);
+      preferredLanguageSaveSuccessDispatch(selectedLanguage.current as Locales);
       setSelectedItem(selectedLanguage.current);
+      showModal(
+        <AlertModal
+          message={I18n.t("profile.main.pagoPaEnvironment.alertMessage")}
+        />
+      );
+      IOToast.success(
```

Comment:
Since we are in a functional component, wouldn't it be better to use the `useIOToast` hook instead of `IOToast`?

Reply:
Good catch! Thank you!

Reply:
Fixed here 926db2e6e4fbb746edd4c0bbbe4b79c524c6bb9c